framer/Animation.coffee

```diff
@@ -33,69 +33,48 @@ evaluateRelativeProperty = (target, k, v) ->
 	return target[k] + (sign + 1) * number if sign
 	return +number
 
-# _runningAnimations = []
-
-# Todo: this would normally be BaseClass but the properties keyword
-# is not compatible and causes problems.
 class exports.Animation extends BaseClass
 
-	constructor: (options={}) ->
-
-		options = Defaults.getDefaults "Animation", options
+	constructor: (parameters={},options={}) ->
+		_.defaults(options, parameters.options)
 
-		super options
+		delete parameters.options
+		@options = _.cloneDeep Defaults.getDefaults "Animation", options
+		super parameters
+		@layer = parameters.layer ? null
```

Comment:
No, because I'd like to be able to clone an animation and set its layer to some other layer before starting it